tests/timeDemo.cc

```diff
@@ -0,0 +1,162 @@
+#include <iostream>
+
+#include <cracl/ublox/m8.hpp>
+
+int main(int argc, char* argv[])
+{
+  using namespace cracl;
+
+  m8 x("/dev/ttyACM0");
+
+  // Disable NMEA messages on uBlox
+  x.disable_nmea();
+
+  //GLO Demo
+  std::cout << "TIMEGLO Demo" << std::endl;
+  x.ubx_send("NAV", "TIMEGLO");
+  sleep(2);
+
+  auto nav_timeglo = x.fetch_ubx("NAV", "TIMEGLO");
+
+  if (ubx::nav::timeglo::type(nav_timeglo))
+  {
+    ubx::nav::timeglo parsed = ubx::nav::timeglo(nav_timeglo);
+
+    std::cout << "iTOW: " << parsed.iTOW() << "ms" << std::endl;
+    std::cout << "TOD: " << parsed.TOD() << "s" << std::endl;
+    std::cout << "fTOD: " << parsed.fTOD() << "ns" << std::endl;
+    std::cout << "Nt: " << parsed.Nt() << "days" << std::endl;
+    std::cout << "N4: " << (int)parsed.N4() << std::endl;
```

Comment:
please use C++ style casts instead of C style
```suggestion
    std::cout << "N4: " << static_cast<int>(parsed.N4()) << std::endl;
```

Reply:
(same for other casts too)